management/ia/rules_test.go

```diff
@@ -163,11 +200,100 @@ func TestRulesAPI(t *testing.T) {
 			_, err = client.UpdateAlertRule(params)
 			pmmapitests.AssertAPIErrorf(t, err, 404, codes.NotFound, "Failed to find all required channels: [%s].", unknownChannelID)
 		})
+
+		t.Run("wrong parameter", func(t *testing.T) {
+			cParams := createAlertRuleParams(templateName, channelID, "param1", dummyFilter)
+			rule, err := client.CreateAlertRule(cParams)
+			require.NoError(t, err)
+			defer deleteRule(t, client, rule.Payload.RuleID)
+
+			params := &rules.UpdateAlertRuleParams{
+				Body: rules.UpdateAlertRuleBody{
+					RuleID:   rule.Payload.RuleID,
+					Disabled: false,
+					Params: []*rules.ParamsItems0{{
+						Name:  "unknown parameter",
+						Type:  pointer.ToString("FLOAT"),
+						Float: 21,
+					}},
+					For:          "10s",
+					Severity:     pointer.ToString("SEVERITY_ERROR"),
+					CustomLabels: map[string]string{"foo": "bar", "baz": "faz"},
+					Filters: []*rules.FiltersItems0{{
+						Type:  pointer.ToString("EQUAL"),
+						Key:   "param1",
+						Value: "21",
+					}},
+					ChannelIds: []string{channelID, newChannelID},
+				},
+				Context: pmmapitests.Context,
+			}
+			_, err = client.UpdateAlertRule(params)
+			pmmapitests.AssertAPIErrorf(t, err, 400, codes.InvalidArgument, "Unknown parameters [unknown parameter].")
+		})
+
+		// t.Run("missing parameter", func(t *testing.T) { TODO: PMM-7279 fix once templates will support parameters without default value.
```

Comment:
let's remove this test or use `t.Skip` instead of commenting it

Reply:
Also we have expected failure mechanism which can be used.

Reply:
Done, added `Skip`